data/json/recipes/chem/other.json

```diff
@@ -26,11 +27,12 @@
     "difficulty": 2,
     "time": 12000,
     "autolearn": true,
-    "qualities": [ { "id": "CHEM", "level": 1 } ],
+    "qualities": [ {
+        "id": "CHEM",
+        "level": 1
+      }
+    ],
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
-    "components": [
-      [ [ "water", 1 ], [ "water_clean", 1 ] ],
-      [ [ "bleach", 1 ], [ "lye_powder", 100 ], [ "bone", 20 ], [ "bone_human", 20 ], [ "bone_tainted", 40 ], [ "cured_hide", 2 ], [ "pine_bough", 10 ], [ "raw_dandelion", 20 ], [ "biollante_bud", 1 ], [ "ant_egg", 1 ], [ "slime_scrap", 1 ] ]
-    ]
+    "components": [ [ [ "water", 1 ], [ "water_clean", 1 ] ], [ [ "bleach", 1 ], [ "lye_powder", 100 ], [ "bone", 20 ], [ "bone_human", 20 ], [ "bone_tainted", 40 ], [ "cured_hide", 2 ], [ "pine_bough", 10 ], [ "raw_dandelion", 20 ], [ "biollante_bud", 1 ], [ "ant_egg", 1 ], [ "slime_scrap", 1 ] ] ]
```

Comment:
This line is well above tolerable length. Made worse by tons of braces.

Reply:
I already addressed this https://github.com/CleverRaven/Cataclysm-DDA/pull/21612#pullrequestreview-57416540

Reply:
"Maybe someday" would work if the PR didn't also get rid of the only formatter capable of outputing sane results for such a recipe.
If the formatter is only capable of formatting very simple entries, it is not ready to replace the old, complete formatter. Instead, it should have its separate whitelist, with the more extensive one still being covered by the formatter which can handle them.